requirements.txt

```diff
@@ -0,0 +1,10 @@
+Flask
```

Comment:
It's very cool that you can add in the requirements in a .txt file and then just `flask install requirements.txt` to install them all.